### PR TITLE
FHAC-296:Transaction Timeout Tests

### PR DESCRIPTION
**Pull request by sailajaa**
Added TransactionTimeout Tests
Removed return(space) between alias and name in x12 proxyconfig.xml files.

### CODE REVIEW

**Comment by nareshcgi** [comment]
Any changes made to CORE_X12DSRealTimeProxyConfig.xml and CORE_X12DSGenericBatchProxyConfig.xml except formatting? If no, then revert back those files, unnecessary in this PR.

**Comment by nareshcgi** [comment]
Not able to run the regression suite test..its hanging on the first Test Suite LargeTransactionTimeout --> Document Submission Mock Response.

**Comment by sailajaa** [comment]
Removed retun(space) between alias and name in Core_x12RealTimeProxyConfig.xml and Core_X12DSGenericBatchProxyConfig.xml which was causing to update realtime adapter from secured to unsecred.

**Comment by sailajaa** [comment]
Does it shows any error when its hanging at Mock step.  Ran LargeTransactionTimeout suite few times, could not able to reproduce error scenario.

**Comment by jsmith2** [comment]
Transaction is spelled wrong in the project name.

**Comment by sailajaa** [comment]
Renamed project name and updated testsuite description.

File: Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml [hunk 1]
```diff
@@ -22,8 +22,7 @@ http://www.springframework.org/schema/aop     http://www.springframework.org/sch
     <!--
         Beans defined for adaptercore_x12dsgenericbatchrequest
     -->
-    <alias alias="adaptercore_x12dsgenericbatchrequest"
-           name="adaptercore_x12dsenericbatchrequestwsunsecured" />
+    <alias alias="adaptercore_x12dsgenericbatchrequest" name="adaptercore_x12dsenericbatchrequestwsunsecured" />
```
Comment (christophermay07) [inline Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml:25]: Is there any reason for changing the Proxy Config files?

**Comment by christophermay07** [comment]
Why are AD test cases disabled by default?

**Comment by christophermay07** [comment]
Following-up on Alamelu's comment about Passthrough:  If this test suite requires passthrough, what is the purpose of the "Check Passthru" test step in each test case?

**Comment by christophermay07** [comment]
One last thing:  If you have to make another push, can you pretty-print save the SoapUI project file?

**Comment by sailajaa** [comment]
AD test case can be removed from all three suites, since that is not used by anyone. But, idea to include and disable is, it might be helpful if we want to test in future if required.

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

File: Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml (hunk 1 above)
Comment (sailajaa) [inline Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml:25]: Removed retun(space) between alias and name in Core_x12RealTimeProxyConfig.xml and Core_X12DSGenericBatchProxyConfig.xml which was causing to update realtime adapter from secured to unsecred.

**Comment by nareshcgi** [comment]
Looks good.

**Pull request merged**